tutorials/real-time-analytics-tutorial.rst

```diff
@@ -0,0 +1,153 @@
+        avatar_url text,                                                                         
+        gravatar_id text,                                                                        
+        display_login text                                                                       
+    );
+
+Next, you can create indexes on events data just like you would do in PostgreSQL.
+    
+::
+                                                                                         
+    CREATE INDEX event_type_index ON github_events (event_type);                                                  
+    CREATE INDEX payload_index ON github_events USING GIN (payload jsonb_path_ops);
+
+Distributing tables and loading data
+------------------------------------
+
+We will now go ahead and tell Citus to distribute these tables across the nodes in the cluster. To do so,
+you can run :code:`create_distributed_table` and specify the table you want to shard and the column you want to shard on.
+In this case, we will shard all the tables on :code:`user_id`.                             
+                                                                                          
+::
+    
+    SELECT create_distributed_table('github_users', 'user_id');                                       
+    SELECT create_distributed_table('github_events', 'user_id');                               
+                                                                                          
+Sharding all tables on the user identifier allows Citus to :ref:`colocate <colocation>` these tables together,
+and allows for efficient joins and distributed roll-ups. You can learn more about the benefits of this approach `here <https://www.citusdata.com/blog/2016/11/29/event-aggregation-at-scale-with-postgresql/>`_.
+                                                                                          
+Then, you can go ahead and load the data we downloaded into the tables using the standard PostgreSQL :code:`\COPY` command.
+Please make sure that you specify the correct file path if you downloaded the file to a different location.
+
+::
+                                                                                          
+    \copy github_users from 'users.csv' with csv;                                                     
+    \copy github_events from 'events.csv' with csv;                                                     
+
+
+Running queries
+----------------
+
+Now that we have loaded data into the tables, let's go ahead and run some queries. First, let's check how many users we have in our distributed database.
+
+::
+                                                                                          
+    SELECT count(*) FROM github_users;
+    
+Now, let's analyze Github push events in our data. We will first find the number of distinct commits in push events and then compute the total number of commits by each minute.
```

Comment:
On reading this description, I thought that the query will have some DISTINCT clause or that we would do something explicitly to find the number of distinct commits before computing the total. Maybe change to something like:-

"We will first compute the total number of commits by each minute by using the number of distinct commits in each push event."